openstack/CleanWlmDatabase/historical_data_cleanup/DeleteHistoricalRecords.py

```diff
@@ -0,0 +1,123 @@
+import os
+import time
+import configparser
+import re
+import mysql.connector
+
+# Get records to be deleted using select query
+# Write those records into file
+# Run delete on those records
+
+batchSize=1000
+
+# check if log directory is present, if not create one
+LOG_DIR="./LOGS"
+if not os.path.exists(LOG_DIR):
+   os.mkdir(LOG_DIR)
+
+# Create Logfile
+timestr = time.strftime("%Y%m%d-%H%M%S")
+LOG_FILE_NAME="deletehistoricalrecord-" + timestr + ".log"
+LOG_FILE_PATH=LOG_DIR + "/" + LOG_FILE_NAME
+logfile = open(LOG_FILE_PATH,"w")
+
+# DB connection 
+## Grab fields from the conf file
+config = configparser.ConfigParser()
+config.read('/etc/workloadmgr/workloadmgr.conf')
+mysql_conn = config['DEFAULT']['sql_connection']
+db_string = re.split('mysql://|=|@|/|:|\?', mysql_conn)
+
+## Edit these 4 variable manually if you want to change any of them
+db_user = db_string[1]
+db_pass = db_string[2]
+db_host = db_string[3]
+db_db = db_string[4]
+
+my_connect = mysql.connector.connect(user=db_user, password=db_pass, host=db_host, database=db_db, connection_timeout=600 )
+# Set mysql connection timeout to 10 min
+cursor = my_connect.cursor(dictionary=True)
+
+
+def getRecords(batchSize):
+    try:
+       filepath="./select_queries/{}".format(table)
```

Comment:
I see that we are using a global variable name `table` which is basically getting defined during an iteration.
Using that as a global variable in any method is not a good practice.
A better way is to pass that parameter to the method where it is getting used.
So this method should be defined as 
```
def getRecords(tableName, batchSize):
    try:
        filepath="./select_queries/{}".format(tableName)
        
        ...
```

Reply:
pls take care of this where we are using `table` variable as a global var